libs/platform/ebpf_platform.h

```diff
@@ -39,11 +39,14 @@ extern "C"
 
 #define EBPF_HASH_TABLE_NO_LIMIT 0
 
-#define ebpf_assert_success(x)                      \
-    do {                                            \
-        ebpf_result_t _result = (x);                \
-        ebpf_assert(_result == EBPF_SUCCESS && #x); \
-    } while (0)
+#define ebpf_assert_success(x)                                     \
+    _Pragma("warning(push)") _Pragma("warning(disable : 4189)") do \
+    {                                                              \
+        ebpf_result_t _result = (x);                               \
+        ebpf_assert(_result == EBPF_SUCCESS && #x);                \
```

Comment:
Maybe something similar as above comment here also.

Reply:
Same as above, with the addition of constrains of it being a macro.

Reply:
Resolving as per the comment above.